docs/CONTRIBUTING/Dependencies.md

```diff
@@ -54,6 +54,27 @@ scl enable devtoolset-7 bash
 
 And build within the resulting bash shell, which has the tools first in PATH.
 
+Debian 9+ (stretch):
+```
+sudo apt-get install libx11-dev libx11-xcb-dev mesa-common-dev libgl1-mesa-dev libxcb-keysyms1-dev cmake python3-dev bison autoconf automake libpcre3-dev
+```
+
+For qrenderdoc:
+```
+sudo apt-get install qt5-qmake libqt5svg5-dev libqt5x11extras5-dev 
```

Comment:
Can you combine this with the `apt-get` line above? All the other distribution instructions list all dependencies together. If a user really wants to avoid building qrenderdoc they can drop the Qt packages from the command. Or if you prefer you could keep the Qt apt-get command separate but instead combine this section with the Ubuntu instructions (since it's the same for both).

The Ubuntu instructions should really be updated either way as they haven't been touched in a couple of years. We'll still have to keep supporting 16.04 for a while yet, but since most people building renderdoc will probably be on a new enough version we can put the straightforward instructions front and centre. Similar to Debian jessie/stretch.